mars-sim-core/src/main/java/com/mars_sim/core/person/ai/CacheCreator.java

```diff
@@ -0,0 +1,89 @@
+package com.mars_sim.core.person.ai;
```

Comment:
Can we create a simple unit test to check the selection logic?

Reply:
Have to give some thoughts on how it can be tested. 

@muybien3433 

Here are the [examples](https://github.com/mars-sim/mars-sim/tree/master/mars-sim-core/src/test/java/com/mars_sim/core/person/ai/task) of the tests that we'll modify after the change is made.